### PR TITLE
[COST-2986] Add `CACHE_TIMEOUT` to deploy config

### PR DESCRIPTION
**Pull request by samdoran**
## Jira Ticket

[COST-2986](https://issues.redhat.com/browse/COST-2986)

## Description

This allows us to control the sources API timeout in our deployments, helping out with testing.

Related to #3712.

### CODE REVIEW

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -2904,6 +2906,9 @@ parameters:
 - displayName: RBAC_CACHE_TIMEOUT
   name: RBAC_CACHE_TIMEOUT
   value: "300"
+- displayName: Middleware Timeout
+  name: CACHE_TIMEOUT
+  value: "300"
```
Comment (adberglund) [inline deploy/clowdapp.yaml:2911]: Let's do 3600 to match the settings file to make this a no-change change in cases where we don't explicitly set this var in app interface.

File: deploy/kustomize/base/base.yaml [hunk 2]
```diff
@@ -424,3 +424,6 @@ parameters:
 - name: RBAC_CACHE_TIMEOUT
   displayName: RBAC_CACHE_TIMEOUT
   value: "300"
+- name: CACHE_TIMEOUT
+  displayName: Middleware Timeout
+  value: "300"
```
Comment (adberglund) [inline deploy/kustomize/base/base.yaml:429]: Samesies

**Review by adberglund — CHANGES_REQUESTED** [review]

File: deploy/clowdapp.yaml (hunk 1 above)
Comment (samdoran) [reply deploy/clowdapp.yaml:2911]: Doh, I meant to do that. 🤦

**Review by adberglund — APPROVED** [review]

**Pull request merged**